doc/release_notes/release_notes_3.0.rst

```diff
@@ -59,8 +85,47 @@ steps to deploy and run the tool.
 Update Configuration Options
 ============================
 
-Complete overhaul of configurator in v3.0...
-
+In v3.0, data in a launch XML are merged into the scenario XML for the new configurator.
+The highly recommended way to use upgrader tool to upgrade a pair of scenario XML and
+launch XML files into a merged file. Then use the ACRN Configurator to open the upgraded
+file for further edits in case any meaningful data is lost. In detail, the following elements
+are refined in the scenario XML file for user-friendly experience.
+
+- ``RDT``
+- ``vUART``
+- ``IVSHMEM``
+- ``Memory``
+- ``virtio devices``
+
+The following elements are added to scenario XML files.
+
+- ``vm.lapic_passthrough``
+- ``vm.io_completion_polling``
+- ``vm.nested_virtualization_support``
+- ``vm.virtual_cat_support``
+- ``vm.secure_world_support``
+- ``vm.hide_mtrr_support``
+- ``vm.security_vm``
+
+The following elements were removed.
+
+- ``hv.FEATURES.NVMX_ENABLED``
+- ``hv.DEBUG_OPTIONS.LOG_BUF_SIZE``
+- ``hv.MEMORY.PLATFORM_RAM_SIZE``
+- ``hv.MEMORY.LOW_RAM_SIZE``
+- ``hv.CAPACITIES.MAX_IR_ENTRIES``
+- ``hv.CAPACITIES.IOMMU_BUS_NUM``
+- ``vm.guest_flags``
+- ``vm.board_private``
+
+In v3.0, we refine the structure of the generated scripts so that PCI functions
+are identified only by their BDF. This change serves as a mandatory step to align
+the way how passthrough devices are configured for pre-launched and post-launched VMs,
+which eventually allows us to present a unified view in the configurator for assigning
```

Comment:
Please make sure the tool name is ACRN Configurator (capital "C") throughout the document.
```suggestion
which eventually allows us to present a unified view in the ACRN Configurator for assigning
```